experiments/probabilistic_domain_adaptation/livecell/punet_source.py

```diff
@@ -26,16 +26,15 @@ def _train_cell_type(args, cell_type, device=DEVICE):
         model=model,
         train_loader=train_loader,
         val_loader=val_loader,
-        loss=DummyLoss(),
         metric=DummyLoss(),
```

Comment:
It's also better if you do this already in the trainer. Then it's not necessary to do it on the user side. (see comment below)